File: assets/samples/brand.alert/ruby/brand_alert_api_example.rb
require 'open-uri'
require 'json'
require 'yaml'	

username = "Your whois api username"
password = "Your whois api password"
term1 = "cinema"
exclude_term1 = 'movie'
exclude_term2 = 'online'

format = "JSON"
url = 'https://www.whoisxmlapi.com/brand-alert-api/search.php?' +
    'term1=' + term1 +
    '&username=' + username +
    '&password=' + password +
    '&output_format=' + format +
    '&exclude_term1=' + exclude_term1
    '&exclude_term2=' + exclude_term2

# Open the resource
buffer = open(url).read

# Parse the JSON result
result = JSON.parse(buffer)

# Print out a nice informative string
puts "JSON:\n" + result.to_yaml + "\n"

File: assets/samples/brand.alert/nodejs/brand_alert_api_example.js
var http = require('https');

const username = 'your whois api username';
const password = 'your whois api password';
const term1 = "cinema";
const exclude_term1 = 'movie';
const exclude_term2 = 'online';
const format = "JSON";

var url = 'https://www.whoisxmlapi.com/brand-alert-api/search.php?' +
    'term1=' + term1 +
    '&username=' + username +
    '&password=' + password +
    '&output_format=' + format +
    '&exclude_term1=' + exclude_term1 +
    '&exclude_term2=' + exclude_term2;

http.get(url, function(response) {
    var str = '';
    response.on('data', function(chunk) { str += chunk; });
    response.on('end', function() { console.log(str); });
}).end();